olive/passes/onnx/vitis_ai_quantization.py

```diff
@@ -213,24 +242,20 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
         # exposed extra options config
         config.update(deepcopy(_exposed_extra_options_config))
         config.update(deepcopy(_extra_options_config))
+
+        # external data config
+        config.update(get_external_data_config())
+        config["execution_providers"] = PassConfigParam(
```

Comment:
It seems the execution provider param is needless? what's the consideration that allow user to override the execution providers passed by engine?

Reply:
We want to use the execution providers to run the calibration in this quantization pass. For example, user can run quantize calibration on GPU to speed up.

Reply:
the execution provider associated with local target will be automatically populated to each pass as accelerator_spec. If you want to override the ep, you can use the execution_providers for engine like https://github.com/microsoft/Olive/blob/main/examples/directml/stable_diffusion/config_vae_decoder.json#LL89C1-L90C1

So, it seems this parameter could be removed.

Reply:
Yes, we agree with you. We are directly using the Pass.accelerator.execution_provider in the codes. The 'config' here is a local variable and will not override the ep.

Reply:
The issue here is that the execution_providers is exposed to the user by _default_config. As the result, the generated document will includes the execution_providers as a parameter. Instead, we could remove the line here and explicitly pass it when invoking quantize_static in line 347

Reply:
The problem here is that we cannot fetch "accelerator_spec.execution_provider" in the "_run_for_config" function in line 347, so we used a local variable to pass the value. Is there any other way to pass the ep value in "_run_for_config"?

Reply:
The same `accelerator_spec` is actually saved in the pass as `self._accelerator_spec` so you can access it in the pass itself. 

If you are doing this, please update this method to return `False`. https://github.com/microsoft/Olive/blob/main/olive/passes/olive_pass.py#L104

Reply:
@jambayk is right.